### PR TITLE
REopt.jl v0.47.0

### PR DESCRIPTION
**Pull request by Bill-Becker**
### Please check if the PR fulfills these requirements
- [ ] CHANGELOG.md is updated
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce?
(Bug fix, feature, docs update, ...)

### What is the current behavior?
(You can also link to an open issue here)

### Does this PR introduce a breaking change?
(What changes might users need to make in their application due to this PR?)

### CODE REVIEW

**Comment by Bill-Becker** [comment]
@hdunham @bpulluta @zolanaj @adfarth You all made commits in the REopt.jl v0.47.0 release. Please add/update what you need to for the API, and when you finished, please drop a note saying so.

@bpulluta I believe I added the new `ProcessHeatLoadInputs`, but let me know if anything else has to be added (I don't think outputs/results need to be added).

**Comment by hdunham** [comment]
> @hdunham @bpulluta @zolanaj @adfarth You all made commits in the REopt.jl v0.47.0 release. Please add/update what you need to for the API, and when you finished, please drop a note saying so.
> 
> @bpulluta I believe I added the new `ProcessHeatLoadInputs`, but let me know if anything else has to be added (I don't think outputs/results need to be added).

No API changes are needed for my Julia PRs.

File: reo/scenario.py [hunk 1]
```diff
@@ -199,33 +199,50 @@ def setup_pv(pv_dict, latitude, longitude, time_steps_per_hour):
...
 
         # Load Profile Boiler Fuel
-        lpbf_space = LoadProfileBoilerFuel(load_type="SpaceHeating",
-            dfm=dfm,
-            time_steps_per_hour=inputs_dict['time_steps_per_hour'],
-            latitude=inputs_dict['Site']['latitude'],
-            longitude=inputs_dict['Site']['longitude'],
-            nearest_city=lp.nearest_city,
-            year=lp.year,
+        lpbf_space = LoadProfileBoilerFuel(
```
Comment (hdunham) [inline reo/scenario.py:202]: Why are there changes in the reo directory, which only affects the discontinued v1-2?

**Comment by zolanaj** [comment]
> @hdunham @bpulluta @zolanaj @adfarth You all made commits in the REopt.jl v0.47.0 release. Please add/update what you need to for the API, and when you finished, please drop a note saying so.
> 
> @bpulluta I believe I added the new `ProcessHeatLoadInputs`, but let me know if anything else has to be added (I don't think outputs/results need to be added).

Confirming no API changes for the changes in v0.47.0.

**Pull request merged**